step-capstone/src/components/Utilities/FoodTimeForm.js

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from "react"
+import {
+    Grid,
+    Box,
+    Slider,
+    Typography
+} from "@material-ui/core"
+
+export default function FoodTimeForm(props) {
+    const MILLISECONDS_PER_MIN = 60000;
```

Comment:
Shouldn't there be a constant in the standard time library for this?

Reply:
I was unable to find one, but there are libraries like [Moment.js](https://momentjs.com/docs/#/durations/milliseconds/) that has conversion functions. Should I switch to using those?